text/0000-template.md

```diff
@@ -0,0 +1,146 @@
+# A Dynamic Configuration Service for the Agent
+
+This proposal adds a configuration service to dynamically configure metric collection and tracing.
```

Comment:
be explicit here saying that this proposal mentions tracing, but leaving it as TBD for the next iteration